students/Kate_Koehn/Lesson02/charges_calc.py

```diff
@@ -0,0 +1,130 @@
+'''
+Returns total price paid for individual rentals
+'''
+import argparse
+import json
+import datetime
+import math
+import logging
+#import os
+
+LOG_FORMAT = "%(asctime)s %(filename)s:%(lineno)-3d %(levelname)s %(message)s"
+
+FORMATTER = logging.Formatter(LOG_FORMAT)
+
+LOGGER = logging.getLogger()
+
+
+def parse_cmd_arguments():
+    """
+    function runs program from the command line
+
+    args: input data file and output file name are required, logging is optional
+    """
+    parser = argparse.ArgumentParser(description='Process some integers.')
+    parser.add_argument('-i', '--input', help='input JSON file', required=True)
+    parser.add_argument('-o', '--output', help='output JSON file', required=True)
+    parser.add_argument('-d', '--debug', help='enable debug logging', required=False)
+    #add in menu in help
+
+    return parser.parse_args()
+
+
+# def check_output_directory(filename):
+#     if not os.path.exists(filename):
+#         os.makedirs()
+
+
+def set_log_level(log_level=0):
+    """
+    function to set level of logging desired from command line when running program
+    :param log_level: 0, 1, 2, or 3
+    :return: returns different level of logging when running the program
+    """
+    if log_level == 0:
+        return
```

Comment:
Notice that all of the logging stuff at the top of the file could be below this return as well. Even less overhead when not logging!